src/main/java/appeng/menu/locator/InventoryItemLocator.java

```diff
@@ -17,16 +19,14 @@
  * Optionally also contains a block position and side in case the menu is to be opened by the item but for a clicked
  * host (i.e. network tool).
  */
-record InventoryItemLocator(
-        int itemIndex,
-        @Nullable BlockPos blockPos) implements MenuItemLocator {
+record InventoryItemLocator(int itemIndex, @Nullable BlockHitResult hitResult) implements ItemMenuHostLocator {
     @Nullable
     public <T> T locate(Player player, Class<T> hostInterface) {
```

Comment:
why did you move the implementation in CuriosItemLocator to the interface, but then keep the nearly identical implementation ere? (the only difference is the logging)